src/clojure_tictactoe/cli/input/input_getter.clj

```diff
@@ -13,19 +13,25 @@
   (get-user-input)
   nil)
 
-(defn parse-move-input
-  ([input]
-   (let [formatted-input (read-string input)]
-     (let [valid-numeral (and
-                           (number? formatted-input)
-                           (< formatted-input 9))]
-       (when valid-numeral
-         formatted-input)))))
+(defn parse-move-input [input]
+  (let [formatted-input (read-string input)]
+    (let [valid-numeral (and
+                          (number? formatted-input)
+                          (< formatted-input 9))]
```

Comment:
This `and` statement can be made into a `let` variable or a function that states what this conditional is checking.

Reply:
Ah whoops. I missed that. The double `let` binding confused me lol. This should be a single `let` binding as well.

Reply:
👍